examples/examples_test.go

```diff
@@ -336,12 +339,19 @@ func TestExampleObjectSchemas(t *testing.T) {
 				t.Logf("skipping : %s/%s\n", path, name)
 				return
 			}
-			if err := latest.Codec.DecodeInto(data, expectedType); err != nil {
-				t.Errorf("%s did not decode correctly: %v\n%s", path, err, string(data))
-				return
-			}
-			if errors := validateObject(expectedType); len(errors) > 0 {
-				t.Errorf("%s did not validate correctly: %v", path, errors)
+			if name == "scheduler-policy-config" {
+				if err := schedulerapilatest.Codec.DecodeInto(data, expectedType); err != nil {
```

Comment:
Maybe add a TODO here saying we should add a validate method for scheduleapi.Policy ?